opacus/tests/grad_samples/group_norm_test.py

```diff
@@ -4,29 +4,35 @@
 import torch
 import torch.nn as nn
 
+from typing import Optional, Tuple, Callable, Union
+
+import hypothesis.strategies as st
+from hypothesis import given, settings
+
 from .common import GradSampleHooks_test
 
 
 class GroupNorm_test(GradSampleHooks_test):
-    """
```

Comment:
nit: Keep this comment?

Reply:
Right. Since I created this from a new file, I must have forgotten to copy and paste it